blank-canvas/style.css

```diff
@@ -69,17 +69,15 @@ Used as block pattern image.
 	margin-top: calc(-1 * var(--global--spacing-vertical));
 }
 
-/* Add some top padding for archive pages. */
+/* Add some top padding */
```

Comment:
I'm pretty sure these padding styles can be removed entirely now. Things seem to work just fine without them.